content/handbook/Interim_processes/decision_making_process.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Interim Decision Making Process
+slug: decision_making_process
+type: docs
+prev: assembly_facilitation
+next: agreements
+weight: 2
+sidebar:
+  open: true
+---
+
+The process outlined below is intended to be replaced by decision-making *agreements* once we have created them (and should be reviewed in October, if not already replaced by then). 
+
+## Key Concepts
+
+MAJOR DECISIONS: Major decisions are those that have high risk and/or permanent outcomes. Initially, our major decisions will be about how we function as a collective, including the principles informing our collective practices and how we make different types of decisions together. Major decisions we will face later (and hope to have an updated decision making process for) include articulating our criteria for determining membership rights and responsibilities, our processes for navigating conflict well together, when and how to acquire property, co-living and multi-site collectivising agreements, etc.,
+
+MINOR DECISIONS: Minor decisions are those with low-risk and easily-reversible outcomes. For now, we are using this for all operational decisions needed to keep the project moving forward. Examples of operational decisions include selecting professionals who can review our T2S model, choosing a location for hosting our next assembly, prioritising the next tasks needed to move the project forward, selecting a process for documenting our decisions, etc.,   
+
+CONSENSUS AGREEMENTS: a [*consensus agreement*](https://www.seedsforchange.org.uk/consensus#flowchart) is a way of reaching agreement by  by discussing all opinions, ideas and concerns and collaborating to find solutions that everyone involved actively support, or at least can live with. While there are many ways of forming consensus agreements - with different degrees of structure and formality as appropriate to the group. The key to making consensus work is for everyone to express their needs and viewpoints clearly, and for the group to use this information to find a solution which builds on the common ground and resolves differences. The stages of a consensus process typically follow some variation of the following stages: introduction; clarification; open discussion (focusing on expressing different perspectives); exploration (focusing on potential solutions); proposal; amendments; agreement-test; and implementation. It is important to remember that consensus agreements function best when everyone shares a common goal and are willing to work together towards it. Finding a solution usually relies on people being flexible about their preferences in order to meet all the cores needs. This requires a lot of trust. 
```

Comment:
Maybe change "While there are many ways of forming consensus agreements" to "While there are many ways of forming agreements by consensus", to keep things verby

Reply:
To be honest, I probably prefer "Consensus Process". It helps us differentiate between the decision we make, and how we make it.